src/v3/src/components/Form/renderers.tsx

```diff
@@ -106,6 +107,10 @@ export default [
     },
     renderer: PhoneAuthenticator,
   },
+  {
+    tester: ({ type }) => type === 'TextWithLink',
```

Comment:
One more update needed here
```suggestion
    tester: ({ type }) => type === 'TextWithHtml',
```

Reply:
Oh yea, fixed this just running jest to update tests appropriately and will push.